app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
+# frozen_string_literal: true
+
+# This is the application controller
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  def index
```

Comment:
This might be better as
```suggestion
  def index; end
```